samples/bluetooth/peripheral_ots/src/main.c

```diff
@@ -134,11 +134,30 @@ static uint32_t ots_obj_read(struct bt_ots *ots, struct bt_conn *conn,
 	return len;
 }
 
+static ssize_t ots_obj_write(struct bt_ots *ots, struct bt_conn *conn,
+			     uint64_t id, const void *data, uint32_t len,
+			     uint32_t offset, uint32_t rem)
```

Comment:
Going with the POSIX mimicry approach would it make sense to consider `size_t len` and `off_t offset`?

Reply:
To be clear: this may or may not make sense. Using the POSIX types you abstract away from protocol details (e.g. if the protocol has 32-bit integers). So you should do this only if you think it makes sense to abstract away from protocol details.

Reply:
I think since the maximum size of the object is also a `uint32_t` and that is part of the protocol, it makes sense to keep these types as `uint32_t` to keep things aligned.

Reply:
I have changed the types to `size_t` and `off_t`.